src/panels/config/integrations/ha-config-entries-dashboard.ts

```diff
@@ -393,6 +402,15 @@ export class HaConfigManagerDashboard extends LitElement {
         right: auto;
         left: 16px;
       }
+      .discovered {
+        padding: 8px 0;
+      }
+      .discovered .config-entry-row {
+        margin: 2px 0;
+      }
+      .discovered img {
+        width: 36px;
```

Comment:
Why not the same width as the integrations?

Reply:
Seems to take up quite a bit of space, but I can remove that so they match

![Screenshot_20200415_152624](https://user-images.githubusercontent.com/28114703/79348883-7ddb4780-7f2d-11ea-97d6-2f172cf99e9a.png)